src/cogs/jaegeraccounts.py

```diff
@@ -32,7 +32,7 @@ def embed(self):
 
     @property
     def is_booked(self):
-        if self.last_booked_to:
+        if self.last_booked_to and self.last_booked_to < datetime.datetime.now(tz=self.last_booked_to.tzinfo):
```

Comment:
`self.last_booked_to` can be None, good idea otherwise.
I think we could nest that.

Reply:
Well, it is an and statement. If first condition is none, it evals to false and doesn't check furhter

Reply:
So it just stops the eval? Didnt know that, sry